packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx

```diff
@@ -367,7 +367,7 @@ export class GroupedListSection extends React.Component<IGroupedListSectionProps
         groupIndex={subGroupIndex}
         groupNestingDepth={nestingDepth}
         groupProps={groupProps}
-        headerProps={headerProps}
+        headerProps={{ ...headerProps, groupLevel: nestingDepth }}
```

Comment:
This in-line Object spread will be a new object reference each render pass and cause a React re-render.

Reply:
better to drop it into a const and then pass it in?

Reply:
changed